emis/.env

```diff
@@ -1,9 +0,0 @@
-DATABASE_NAME=lucia_education
-DATABASE_USER=root
-DATABASE_PASSWORD=root
-DATABASE_HOST=127.0.0.1
-DATABASE_PORT=8889
-DEV_EMAIL_FILE_PATH=/Applications/MAMP/htdocs/tmp/email
-LOG_DIRECTORY=/Applications/MAMP/logs/emis-pas
-LOG_APP_DEBUG=TRUE
-
```

Comment:
.env file should not be in source control as passwords and settings could easily be checked in. It also provides cleaner production deployments (doing a git pull without having to readjust the .env file).

Reply:
Good idea